kolibri/content/content_db_router.py

```diff
@@ -80,7 +80,7 @@ def get_content_database_connection(alias=None):
             raise KeyError("Content DB '%s' is empty!!" % alias)
 
     # if possible, attach the default database to the content database connection to enable joins
-    _attach_default_database(alias)
+    # _attach_default_database(alias) # Fix 1818.1 db locked and cannot be removed on Windows
```

Comment:
All this "ATTACH DATABASE" stuff turns out to be completely irrelevant.  I've since reverted the commit your comment refers to and tested again, db file deletion works just fine.

Sorry for the false direction... I was reading a lot of horror stories from stackoverflow about windows sqlite3 locking cases, and this seemed like a good technological uncertainty to chase...